_i18n/en.yml

```diff
@@ -1,26 +1,24 @@
 ---
-global:
-  brazilian portuguese: Português
-  english: English
-header:
+sections:
   stories: Stories
   explore: Explore
   about: About
-support:
+  faq: FAQ
+  in_other_language: "[Em Português]"
   fund_us: Support the project
 home:
-  about: Artificial Intelligence for social control of public administration
-  about_rosie: Operação Serenata de Amor created <a href="https://twitter.com/RosieDaSerenata" class="purple">Rosie</a>  - An artificial intelligence capable of analysing each claim for reimbursement from congresspeople and of identifying it’s illegal probability.
-  last_stories: Last stories
+  tagline: Artificial Intelligence for social control of public administration
+  about: Um projeto aberto que usa ciência de dados (e as mesmas tecnologias utilizadas por gigantes como Google, Facebook e Netflix) para fiscalizar gastos públicos e compartilha as informações de forma acessível a qualquer pessoa.
```

Comment:
@cuducos Can you add an English version?

Reply:
Done in a29de4c